aiosmtplib/errors.py

```diff
@@ -78,6 +79,16 @@ def __init__(self, code: int, message: str) -> None:
         self.args = (code, message)
 
 
+class SMTPConnectResponseError(SMTPResponseException, SMTPConnectError):
+    """
+    An error occurred while connecting to the SMTP server.
```

Comment:
How about "The SMTP server returned an invalid response code after connecting." or similar so it's a bit more clear in the docs?

Reply:
Done!